build/helper/parameter_usage_options.py

```diff
@@ -20,10 +20,10 @@ class ParameterUsageOptions(AutoNumber):
     '''For declaring an init method in Session'''
     SESSION_NUMPY_INTO_METHOD_DECLARATION = ()
     '''For declaring a Session method that uses numpy arrays.'''
-    LIBRARY_INTERPRETER_NUMPY_INTO_METHOD_DECLARATION = ()
-    '''For declaring a LibraryInterpreter method that uses numpy arrays.'''
-    LIBRARY_INTERPRETER_NUMPY_INTO_METHOD_CALL = ()
-    '''For calling into a LibraryInterpreter method (from Session) that uses numpy arrays.'''
+    INTERPRETER_NUMPY_INTO_METHOD_DECLARATION = ()
+    '''For declaring a **Interpreter method that uses numpy arrays.'''
+    INTERPRETER_NUMPY_INTO_METHOD_CALL = ()
+    '''For calling into a **Interpreter method (from Session) that uses numpy arrays.'''
```

Comment:
What's with the **?

Reply:
Why isn't it just one *? Is this a convention that I'm not familiar with?

Reply:
No convention, just how my brain works, I guess.